OneCommunityGlobal/HighestGoodNetworkApp
# Ramakrishna create different view for summary

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/624aacad-28ca-4c01-a0b8-a7173e4df90b)


## Related PRS (if any):
Please use latest backend development branch 
…

## Main changes explained:
Update the file src/components/SummaryBar/SummaryBar.jsx to incorporate a different view for users exempt from summary submission
Update the file src/components/BMDashboard/BMDashboard.test.jsx this file changes is not related to this feature just added because of one test case failing 
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. login as mentor user 
5. User should able to see summary instead of  "!summary" and also user should able to see "You don’t need to complete a weekly summary, but you still can. Click here to submit it."  instead of  "You still need to complete the weekly summary. Click here to submit it."  and rest of the operations regarding to summary should work as expected  as before
6. Now login as admin/owner and go to anyone of you're choice  user management-> user profile-> volunteering times-> and change weekly summary options to Not required and login to those user and check step 5 is working expected for more detail please follow the demo video.


## Screenshots or videos of changes:
Before changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/98c2a69e-d445-442d-973b-b72c204068dc)
After Changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/56d7870e-516d-4d24-95ac-2416e450803c)

Video Demo:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/2f6fe26e-8738-4edf-9958-c1b83cb85930
